ext/RMagick/extconf.rb

```diff
@@ -136,18 +136,24 @@ def configure_compile_options
         `identify -version` =~ /Version: ImageMagick (\d+\.\d+\.\d+)-+\d+ /
         abort 'Unable to get ImageMagick version' unless Regexp.last_match(1)
         $magick_version = Regexp.last_match(1)
-        search_paths_for_library_for_mingw unless have_library('CORE_RL_magick_')
+
+        dir_paths = search_paths_for_library_for_windows
+        $CPPFLAGS = %(-I"#{dir_paths[:include]}")
+        $LDFLAGS = %(-L"#{dir_paths[:lib]}")
+
+        have_library('CORE_RL_magick_')
```

Comment:
Does this `have_library` do anything here? it looks like we're returning the line below, so not sure whether we need this.

Reply:
Seems that `have_library('CORE_RL_magick_')` check whether specified library is existed and generate compiler flag to link the library.

Reply:
Ah, I see. I thought maybe it just returned `true` or `false`.